src/components/InstructionSection/InstructionCopy.tsx

```diff
@@ -0,0 +1,36 @@
+import { InstructionText } from 'components/Text'
+import { toast } from 'react-toastify'
+import { useCallback } from 'preact/hooks'
+import Button from 'components/Button'
+import TokenScreenParams from 'models/TokenScreenParams'
+
+export function InstructionCopyButton({ token }: TokenScreenParams) {
+  const onCopy = useCallback(async () => {
+    if (!token) return
+    try {
+      await navigator.clipboard.writeText(token)
+      toast('Token copied 🎉')
+      console.log('Copied to clipboard')
+    } catch (e) {
+      console.error('Failed to copy to clipboard', e)
```

Comment:
We can should toast this as well, let's make `handleError({toast: 'Failed to copy to clipboard', e})`

Reply:
There is no need to create a separate function for that. But anyway add a toast on error too